docs/docs/usage/execution_modes.md

```diff
@@ -0,0 +1,29 @@
+---
+sidebar_position: 110
+sidebar_label: Execution modes
+title: Execution modes
+---
+
+## Overview
+
+| Mode | Performance | Dynamic Module Loading | Requirements | Output Format | Ideal Use Case |
+|---|---|---|---|---|---|
+| **Interpreter** | 🐢 Slow | ✅ Supported | None | None - fully interpreted | Default mode; highly portable; suitable for development and environments requiring dynamic loading. |
+| **Runtime Compilation** | 🐇 Fast | ✅ Supported | Requires reflection and ASM dependency | In-memory Java Bytecode | Enhanced performance; suitable when dynamic loading is needed and the usage of reflection is fine. |
+| **Build time Compilation** | 🐇 Fast | ❌ Not Supported | Build-time tools(e.g., Maven or Gradle plugins) | Plain Java Bytecode | Optimal performance; no dynamic loading; ideal for production with static modules. |
```

Comment:
👍 makes sense

Reply:
I pushed to the main repo to have the Cloudflare preview, but it didn't worked :sweat:

Reply:
no, it works: https://update-compiler-docs.chicory.pages.dev/docs/usage/execution_modes
but the bot have not replied .. .for reasons ...